### DISCUSSION TITLE
Configurable regular maintenance window

### DISCUSSION THREAD

**Discussion by lukasbestle** [upptime/upptime#230] [Ideas]
I have a service that is expected to be down for a few minutes each night.

It would be great to have a site-specific configuration option for this so that Upptime can skip checking the affected site in that regular maintenance window.

**Comment by AnandChowdhary** [comment]
Great idea! We're discussing this in https://github.com/upptime/upptime/discussions/122, so I'm closing this as a duplicate.

**Reply by lukasbestle** [reply]
I'd say it's a different idea. #122 is about scheduled maintenance issues (like "there will be maintenance next Thursday between 9-10 PM"). This idea is about a daily time slot where checking is disabled.

**Reply by AnandChowdhary** [reply]
Aah, I misunderstood you. Makes sense!

**Comment by modem7** [comment]
Would definitely be useful. 

I have a  backup windows at 0500 each day which necessitates some services going down for a few minutes, so having this feature would definitely be utlised heavily.

Maybe something like:

```
day: mon, tues, thurs, sat
task: Backup <--- Label for status page
start: T13:00:00.220Z
end: T14:00:00.220Z
expectedDown: google, hacker-news
```

With a "Scheduled" tag?

**Reply by pluim003** [reply]
It's an old thread, but I was wondering the same as I also have the same that one of my internal websites (actually most of them) is down for a few minutes due to a backup. Would be nice if we could indeed use a daily expectedDown.
Upvoted it.

**Comment by obfuscatedgenerated** [comment]
Old discussion but would be useful. I have a server designed to restart for a few minutes at 5AM UTC, so this would be great. For now I've had to set the cron job to skip xx:00 times which isn't ideal.

**Reply by AnandChowdhary** [reply]
Agreed! Would you be up to make a PR to ad a recurring feature to the existing https://github.com/orgs/upptime/discussions/122#discussioncomment-401551?
